components/pictogram/tests/unit/pictograms.spec.tsx

```diff
@@ -0,0 +1,72 @@
+import { GEL } from '@westpac/core';
+import { render } from '@testing-library/react';
+import wbc from '@westpac/wbc';
+import { overridesTest } from '../../../../helpers/tests/overrides-test.js';
+import { nestingTest } from '../../../../helpers/tests/nesting-test.js';
+import * as pictograms from '@westpac/pictogram';
+import { Pictogram, PictogramProps } from '../../src/Pictogram.tsx';
+
+overridesTest({
+	name: 'pictogram',
+	overrides: ['Pictogram'],
+	Component: (props: any) => <Pictogram copyrightYear="2023" {...props} />,
+});
+
+nestingTest({
+	name: 'pictogram',
+	Component: (props: any) => <Pictogram {...props} />,
+});
+
+function SimplePictogram(props: PictogramProps) {
+	return (
+		<GEL brand={wbc}>
+			<Pictogram data-testid="test-pictogram" {...props} />
+		</GEL>
+	);
+}
+
+describe('Pictogram components', () => {
+	test('should render base Pictogram component', () => {
+		const { container } = render(<SimplePictogram copyrightYear="2020" />);
+		expect(container).toBeInTheDocument();
+	});
+	// Tests every Pictogram in the library to see if it renders correctly
+	for (let [key, val] of Object.entries(pictograms)) {
+		// need to create new variable with expected typing and check if val is function for TS
+		let typedVal: Function;
+		if (typeof val === 'function') {
+			typedVal = val;
+		}
+		test(`should render ${key} with testing prop options`, () => {
+			const TestPictogram = (props: PictogramProps) => {
+				return typedVal({ ...props });
+			};
+			const DefaultTestPictogram = () => {
+				return typedVal({} as PictogramProps);
```

Comment:
As per above comment, regarding the missing copyrightYear prop, but in DefaultTestPictogram this time

Reply:
Discussed on Teams

Reply:
From Team: This is forcing the component to use default props to test the default props as there were issues where it was expecting the required props even with spreading props so I couldn't test defaults unless I did what I did here.